pom.xml

```diff
@@ -93,6 +93,33 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
as we put flatten plugin here it will also be applied to root multi module project.
And we don't really need to do that... 

But the way to avoid that is to make difference between root multi module and parent module. (which is rarely done but allow much more flexbility in build config)